tests/base_tests.py

```diff
@@ -222,22 +222,27 @@ def run_sql(
         query_limit=None,
         database_name="examples",
         sql_editor_id=None,
+        select_as_cta=False,
+        tmp_table_name=None,
     ):
         if user_name:
             self.logout()
             self.login(username=(user_name or "admin"))
         dbid = self._get_database_by_name(database_name).id
+        json_payload = dict(
+            database_id=dbid,
+            sql=sql,
+            client_id=client_id,
+            queryLimit=query_limit,
+            sql_editor_id=sql_editor_id,
+        )
```

Comment:
While the old example used `dict()`, I think we prefer to use `{}` notation.

Reply:
Personally I don't feel strongly about this. If a linter can't enforce it, I'd say either approaches are fine.

Reply:
I prefer {} as well, if both are fine I'll stick with {} rather than dict, thanks